oaknational/Oak-Web-Application
# Feat/lesq 637/legacy remove duplicate description [LESQ-637]

## Description

Music year: 1976

- remove the pupil lesson outcome from the lesson header when it is included in key learning points
- only applies to EYFS currently, but will apply to legacy when migrated

## How to test
1. Go to https://deploy-preview-2247--oak-web-application.netlify.thenational.academy/teachers/programmes/citizenship-secondary-ks4-l/units/how-well-does-the-media-hold-those-in-power-to-account-63c4/lessons/what-is-the-role-of-the-media-in-a-democracy-cmw32c
3. You should see no pupil lesosn outcome in the header (same as prod)
4. Go to https://deploy-preview-2247--oak-web-application.netlify.thenational.academy/teachers/programmes/combined-science-secondary-ks4-higher-aqa/units/measuring-waves/lessons/transverse-waves
5. You should see the pupil lesson outcome in the header (same as prod)

## Screenshots

How it used to look (delete if n/a):
<img src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/50bc261a-b12b-476f-ad90-e6a535d60019" width=500>

How it should now look:
<img src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/d6135ae7-629b-4876-8706-efa13708b869" width=500>
